### DISCUSSION TITLE
Support for Social Login Providers

### DISCUSSION THREAD

**Discussion by rishavs** [luckyframework/lucky#1709] [Feature requests & Ideas]
Would like to add social login capabilities to my Lucky site. The preferred providers to start off would be

- Google
- Apple
- Github
- Facebook

**Comment by matthewmcgarvey** [comment]
While Lucky doesn't provide that functionality, there is a shard for it with documentation on how to use it in a Lucky app. https://github.com/msa7/multi_auth#lucky-integration-example

If you do get it working, definitely let us know how it went!
